var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -130,6 +137,13 @@ class RocmOpenmpExtras(Package):
             placement='rocm-device-libs',
             when='@' + ver)
 
+        resource(
```

Comment:
I would prefer the resources for master to be outside of this for loop. Also please move the resources outside/above the for loop after the depends_on for master version.

Reply:
Done